### PR TITLE
feat: improve thorchain status polling reliability

### LINKED ISSUE #5656: Withdraw modal for ThorChain Saver Vaults not updating to Complete once TX has confirmed on the blockchain
### Overview

When doing a withdraw from a ThorChain saver, the modal does not update to say the tx is completed even after it has confirmed on the blockchain. (Noticed for all chains except ETH) 

### References and additional details

1) Log into app.shapeshift and connect to wallet of choice
2) Navigate to DeFi page and select a saver vault (not ETH)
3) Select withdraw and follow prompts to broadcast the tx
4) View the tx on chain and note that once its confirmed, the modal does not update
image
image
image

### Acceptance Criteria

Modal should update to "Transaction Confirmed" once the tx shows successful on the blockchain

### Need By Date

_No response_

### Screenshots/Mockups

_No response_

### Ownership

- [X] If my bounty needs engineering or needs product I have added the respective labels on the right
- [X] As the sponsor of this bounty I will review the changes in a preview environment (ops/product) or review the PR (engineering)

### Estimated effort

_No response_

### Sponsor / Stakeholder

_No response_

### Bounty Hunters

- [Join our discord](https://discord.gg/shapeshift)
- Include an expected timeline for you to complete work in the work plan when you apply for this bounty!
- Please refer to [this link](https://shapeshiftdao.gitbook.io/getting-started/how-to-contribute/bounties#resources-for-new-contributors-and-bounty-hunters) for some basic info
- Please ***do not*** start work on this issue until you are approved in Gitcoin.

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

THOR Tx status detection can currently take a large amount of time - which is *not* a bug, outbound Txs sometimes take a very long time to get fired.
It looks like the API recently had some (at least as far as we're concerned) breaking changes where `outbound_signed` is always present, regardless of the THOR Tx type. As such, relying on its *presence*, or lack thereof, doesn't cut it anymore.
This PR introduces a `skipOutbound` option, which does precisely that, skips outbound signed checks for lending and savers Txs, since these do not rely on an outbound Tx.

Also includes some fixes/improvements for THORChain lending.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #5656

## Testing

- Ensure depositing some collateral into lending spins until the deposit is complete, and the overview screen reflects the newly deposited amount once completed
- Ensure THORChain savers deposits/withdraws update to completed, including for UTXOs

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

### Lending collateral deposit

https://github.com/shapeshift/web/assets/17035424/68284e06-6e48-4a86-9aff-17a754719373

### ATOM deposits

https://github.com/shapeshift/web/assets/17035424/56aec6c9-5e7e-4293-bf83-1f17e01be6f0

### ATOM withdraws

https://github.com/shapeshift/web/assets/17035424/0f2a63a0-9599-4578-89e4-e4c523656b84

### DOGE deposits

https://github.com/shapeshift/web/assets/17035424/781c2287-45dc-4ed6-9010-15fdb9253751

### DOGE withdraws

https://github.com/shapeshift/web/assets/17035424/25f1992f-79e8-4a01-9009-925fce002e76

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5659** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5659?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5659?utm_source=stack-comment).

File: src/pages/Lending/hooks/useLendingPositionData.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const useLendingPositionData = ({ accountId, assetId }: UseLendingPositio
 
   const lendingPositionData = useQuery({
     // 2 minutes before the data is considered stale, meaning firing this query will trigger queryFn
```
Comment (kaladinlight) [inline src/pages/Lending/hooks/useLendingPositionData.tsx:22]: update comment or make generic so changing `staleTime` doesn't invalidate comment each time

File: src/pages/Lending/hooks/useLendingPositionData.tsx [hunk 2]
```diff
@@ -44,6 +44,9 @@ export const useLendingPositionData = ({ accountId, assetId }: UseLendingPositio
       }
     },
     enabled: Boolean(accountId && assetId && poolAssetMarketData.price !== '0'),
+    refetchOnMount: true,
+    refetchInterval: 60_000,
+    refetchIntervalInBackground: true,
```
Comment (kaladinlight) [inline src/pages/Lending/hooks/useLendingPositionData.tsx:49]: informative question: do the refetch params honor `staleTime` or is `staleTime` only relevant to manual query function calls?

**Review by kaladinlight — APPROVED** [review]
untested, but conceptual stamp

File: src/pages/Lending/hooks/useLendingPositionData.tsx (hunk 1 above)
Comment (gomesalexandre) [reply src/pages/Lending/hooks/useLendingPositionData.tsx]: 00b84b40774175505e9f5f07a7ecb5cb45b857aa

File: src/pages/Lending/hooks/useLendingPositionData.tsx (hunk 2 above)
Comment (gomesalexandre) [reply src/pages/Lending/hooks/useLendingPositionData.tsx:49]: As far as I understand it, refetch is just a manual refetch, similar to calling the query at component-level with the hook i.e would still honor `staleTime`.

**Pull request merged**